<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamesW</class>
 <widget class="QDialog" name="GamesW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GamesW{
	border-image: url(:/imagini/b4.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_mwuser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_mwusername">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_mwusername{
color:black;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="GAMESgb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>271</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#GAMESgb{
color:white;
background:rgb(0, 170, 255);
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>GAMES</string>
   </property>
   <widget class="QPushButton" name="pb_joc1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc1{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc1:hover{
	background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc2{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc2:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc3{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc3:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc4{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc4:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc5{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc5:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_joc6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_joc6{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#pb_joc6:hover{
background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FRIENDSgb">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>301</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#FRIENDSgb{
color:white;
background:rgb(0, 170, 255);
border-radius:8px;
}</string>
   </property>
   <property name="title">
    <string>MY FRIENDS</string>
   </property>
   <widget class="QListWidget" name="friendslist">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>211</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#friendslist{
color:white;
background:rgb(255, 170, 0);
border-radius:8px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_search{
color:black;
background:rgb(255, 170, 0);
border-radius:8px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#delete_button{
color:white;
background-color: rgb(170, 0, 0);
border-width:0px;
border-radius:4px;
}
#delete_button:hover{
background-color:#27a9e3;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#add_button{
color:white;
background-color:#66c011;
border-width:0px;
border-radius:4px;
}
#add_button:hover{
background-color:#27a9e3;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
